src/deepsparse/transformers/haystack/utils.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
to conform with repo standards, let's rename to `helpers.py` (reason being to avoid namespace conflicts if a `utils/` dir is added in future)